fuel-vm/src/tests/predicate.rs

```diff
@@ -204,7 +204,7 @@ async fn get_verifying_predicate() {
 async fn execute_gas_metered_predicates(
     predicates: Vec<Vec<Instruction>>,
 ) -> Result<u64, ()> {
-    const GAS_LIMIT: Word = 10000;
+    const GAS_LIMIT: Word = 20000;
```

Comment:
Were some tests running out of gas or something?

Reply:
Yes, because now `min_gas` is higher than before, it fails out of gas. But I decided to revert this change and fix the test configuration instead.